File: Arrays/BasicArray.java
import java.util.Scanner;

public class BasicArray {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    //Declare an array of size 5
    int[] arr = new int[5];

    //input elements
    for(int i = 0; i<arr.length; i++){
      arr[i] = sc.nextInt();
    }

// output elements
    for(int i =0; i < arr.length; i++ ){
     System.out.print(arr[i] + "");
    }

//size of array
    System.out.println("\nSize:" + arr.length);
    sc.close();
      
  }
}

File: Arrays/basic_array.cpp
#include <iostream>
using namespace std;

int main() {
  //declare an array of size 5
  int arr[5] = {1,2,3,4,5}
  
}


for(int i = 0; i<5; i++) {
  cin >> arr[i];
}

//Output elements
for(int i =0; i < 5; i++) {
  cout << arr[i] <<"";
}


//size of array 
int n = sizeof(arr) / sizeof(arr[0]);
cout<< "\nSize:" << n;

return 0;
}
 
